scormxblock/static/js/src/scormxblock.js

```diff
@@ -125,6 +128,50 @@ function ScormXBlock_${block_id}(runtime, element) {
       return false;
     }
 
+    function disableNextModuleArrow() {
+      if (isNewUI()) {
+        disableNextModuleArrowForNewUI();
+      }
+      else {
+        disableNextModuleArrowForOldUI();
+      }
+    }
+
+    function isNewUI() {
+      return $( "body").hasClass("new-theme");
+    }
+
+    function disableNextModuleArrowForOldUI() {
+      var nextModuleLink = $( ".controls .next" );
+      if (isNextLinkAlreadyDisabled(nextModuleLink)) return;
+      nextModuleLink.addClass( "complete-scorm future" );
+      nextModuleLink.attr("href", "#");
+
+      var completionPopup = document.createElement('div');
+      completionPopup.className = "complete-scorm-content";
+      completionPopup.innerHTML = '<i class="fa fa-lock"></i>Complete all content to unlock';
+
+      $(".controls .next .mcka-tooltip").prepend(completionPopup);
+    }
+
+    function disableNextModuleArrowForNewUI() {
+      var nextModuleLink = $( ".controls .right" );
+      if (isNextLinkAlreadyDisabled(nextModuleLink)) return;
+
+      nextModuleLink.addClass( "disable" );
```

Comment:
Please remove the extra space between small parenthesis and commas. `("disable")`.
same on L158